README.md

```diff
@@ -200,6 +207,12 @@ First of all, build the image by executing this command at the root of PyRDP (wh
 docker build -t pyrdp .
 ```
 
+As an alternative we have a slimmer image without the GUI and ffmpeg dependencies:
+
+```
+docker build -f Dockerfile.slim -t pyrdp .
```

Comment:
is it normal to have this same text at two places in the readme?

Reply:
One is in the section to install pyrdp (`docker pull`) and the other is in the section if you want to build pyrdp with docker (`docker build`). Installing is described on top of the README and advanced usage like docker builds is described later in the file. I think it's normal and fine.

Reply:
Can I resolve this?

Reply:
yeah, doing it